Source/WebDriver/socket/HTTPServerSocket.cpp

```diff
@@ -108,7 +108,7 @@ void HTTPRequestHandler::sendResponse(HTTPRequestHandler::Response&& response)
 {
     auto& endpoint = RemoteInspectorSocketEndpoint::singleton();
     auto packet = packHTTPMessage(WTFMove(response));
```

Comment:
We don't need this local variable any more but we can keep it if we think it adds clarity.